frontend/src/services/users/users.service.ts

```diff
@@ -16,9 +20,16 @@ class UsersApi {
     this.#apiPrefix = apiPrefix;
   }
 
-  public getAll(): Promise<UsersGetAllResponseDto> {
+  public getPaginated({
+    page,
+    count,
+  }: EntityPaginationRequestQueryDto): Promise<
+    EntityPagination<UsersGetResponseDto>
+  > {
     return this.#http.load(
-      `${this.#apiPrefix}${ApiPath.USERS}${UsersApiPath.ROOT}`,
+      `${this.#apiPrefix}${ApiPath.USERS}${
+        UsersApiPath.ROOT
+      }?page=${page}&count=${count}`,
```

Comment:
As in future we can also have other requests with querystring - it would be better to enhance load method to take queryString obj and attach it to request url

Reply:
@andanfe  Should I set its type to `EntityPaginationRequestQueryDto`? If we can send some values, other than the ones for pagination (for example - searching), it'll be incorrect. Other option would be making it of `any` type, but I'm not really sure if it is a good idea

Reply:
I think you can type it as `Record<string, unknown>` and make it optional for `HttpOptions`, then check if it exists, you will provide querystring form it; if it's not - leave url as it goes